Automattic/wp-calypso
# Free Photo Library Card: Display Support Article Dialog

#### Changes proposed in this Pull Request

This updates the Free Photo Library card to show the Support Article Dialog, as mentioned by @lancewillett in #40080. 

<img width="1358" alt="Screenshot 2020-03-15 at 18 37 29" src="https://user-images.githubusercontent.com/43215253/76708319-d1b7fe00-66ed-11ea-921f-15d6d08ffc53.png">

#### Testing instructions

1. Go to Customer Home
2. Click "Learn more" within the Free Photo Library card
3. Verify the dialog above appears

@mmtr I'm conscious this goes against the approach you recommended [here](https://github.com/Automattic/wp-calypso/pull/39506#discussion_r383838749), but this seems to be the only way to get this working. There's not many examples of this in Calypso though, is there an alternative?

cc @gwwar
